docs/rackhd/tasks.rst

```diff
@@ -394,6 +394,9 @@ At render time, values are rendered if the exist in the task render context. The
            - **lookups** -- the base http URI for the RackHD api files route (e.g. `http://<server>:<port>/api/current/lookups`)
            - **files** -- the base http URI for the RackHD api files route (e.g. `http://<server>:<port>/api/current/files`)
            - **nodes** -- the base http URI for the RackHD api nodes route (e.g. `http://<server>:<port>/api/current/nodes`)
+   * - file
```

Comment:
I know 'file' has exists in previous codes, but it indeed confuse me when see 'file' means 'file server', so in my opinion it's better to refine it to another meaningful name in future, it's :+1: from me for this doc PR based on previous codes implementation.